sdk/testing/src/test/java/io/opentelemetry/sdk/testing/junit5/OpenTelemetryExtensionTest.java

```diff
@@ -85,7 +85,12 @@ public void exportTraces() {
             trace ->
                 trace
                     .hasTraceId(traceId)
-                    .hasSpansSatisfyingExactly(s -> s.hasName("testa1"), s -> s.hasName("testa2"))
+                    .hasSpansSatisfyingExactly(
+                        s -> s.hasName("testa1"),
+                        s ->
+                            s.hasName("testa2")
+                                .hasParentSpanId(trace.getSpan(0).getSpanId())
```

Comment:
So with current "hasParent" these two check exactly the same (although the second one is a bit shorter). Was the intention here to demonstrate that? :)

Reply:
That was the intent indeed. And apparently I need to check with the non-0 index in the test...